exporter/newrelic/newrelic.go

```diff
@@ -0,0 +1,27 @@
+package newrelic
+
+import (
+	"context"
+	"errors"
+	"log"
+
+	opencensus "github.com/devopsfaith/krakend-opencensus"
```

Comment:
Since we will merge the exporter only for v2, all the KrakenD/Lura dependencies should point to `/v2` (for now in the `v2.0-dev` branch for `krakend-opencensus` and `v2.0.0-dev` branch for `lura`

Reply:
@taik0 It's not clear what this means.  You want us to fork `v2.0-dev` and PR back to that?

Reply:
we are getting the v2.0 ready, so we are merging all the PRs into this new one. In order to that, all the deps should point to lura/v2 and krakend-opencensus/v2